lib/rspec_junit_formatter/rspec3.rb

```diff
@@ -45,6 +45,10 @@ def examples
     @examples_notification.notifications
   end
 
+  def error_count
+    reports_errors_outside? ? @summary_notification.errors_outside_of_examples_count : 0
```

Comment:
How about:

```suggestion
    # Introduced in rspec 3.6
    if @summary_notification.respond_to?(:errors_outside_of_examples_count)
      @summary_notification.errors_outside_of_examples_count
    else
      0
    end
```

Reply:
That works for me.  I will make that change and push it.